Modal/js/script.js

```diff
@@ -1,18 +1,18 @@
-// Get the modal
 var modal = document.getElementById("myModal");
 
-// Get the button that opens the modal
 var btn = document.getElementById("myBtn");
-
-// Get the <span> element that closes the modal
+var click_btn = document.getElementsByClassName("btn-click")[0];
+var text_display = document.getElementsByClassName("text-display")[0];
 var span = document.getElementsByClassName("close")[0];
 
-// When the user clicks the button, open the modal
 btn.onclick = function () {
   modal.style.display = "block";
 };
+click_btn.onclick = function () {
```

Comment:
a new line in btw line 10 and 11 should make code clean and more redable

Reply:
done